core/runtime.rs

```diff
@@ -1586,6 +1591,38 @@ impl JsRuntime {
 
     Ok(())
   }
+
+  fn drain_nexttick(&mut self) -> Result<(), AnyError> {
+    let state = Self::state(self.v8_isolate());
+
+    let js_nexttick_cb_handle = match &state.borrow().js_nexttick_cb {
+      Some(handle) => handle.clone(),
+      None => return Ok(()),
+    };
+
+    if !state.borrow().has_tick_scheduled {
+      let scope = &mut self.handle_scope();
+      scope.perform_microtask_checkpoint();
+    }
+
+    // TODO(bartlomieju): Node also checks for absence of "rejection_to_warn"
```

Comment:
Currently there's no handling of "unhandledRejection" in Deno so this conditional is skipped, I'm sure it will be problematic when trying to enable more of Node tests

Reply:
Side effect is that `process.nextTick()` doesn't behave properly if error is thrown during its callback